packages/renderer/src/lib/image/RunImage.svelte

```
@@ -293,42 +270,44 @@ async function startContainer(): Promise<void> {
+  const CapAdd = options.security.capAdds.filter(cap => cap);
+  const CapDrop = options.security.capDrops.filter(cap => cap);
 
-  const ExtraHosts = extraHosts.filter(host => host.host && host.ip).map(host => `${host.host}:${host.ip}`);
+  const ExtraHosts = options.networking.extraHosts
+    .filter(host => host.host && host.ip)
+    .map(host => `${host.host}:${host.ip}`);
 
-  const Privileged = privileged;
+  const Privileged = options.security.privileged;
 
   let NetworkMode;
```

Biome (2.5.1)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)